qmake/appimageupdaterbridge.pri

```diff
@@ -13,5 +13,10 @@ unix {
 		DEFINES += USE_APPIMAGE_UPDATER_BRIDGE
 		CONFIG *= link_pkgconfig
 		must_pkgconfig(AppImageUpdaterBridge)
+		message(Will be using AppImageUpdaterBridge)
+	}
+} else {
+	APPIMAGE_UPDATER_BRIDGE_ENABLED {
+		error(You cannot use AppImageUpdaterBridge for this type of build)
```

Comment:
I'd make the error message state that the problem is that AppImageUpdaterBridge only works on Linux. That might be more helpful and more specific than stating it is not available "for this kind of build" :)